comixed-frontend/src/app/library/actions/library.actions.ts

```diff
@@ -151,6 +151,7 @@ export class LibraryConvertComics implements Action {
       archiveType: string;
       renamePages: boolean;
       deletePages: boolean;
+      deleteOriginalComic: boolean;
```

Comment:
Could we rename this parameter to something shorter, like "deleteOriginal"?

Reply:
ok